File: order.py
"""
order.py

Defines the Order class used by the Online Food Delivery Order Management System.
"""

class Order:
    def __init__(self, order_id: str, customer_name: str, food_item: str, quantity: int):
        self.order_id = order_id
        self.customer_name = customer_name
        self.food_item = food_item
        self.quantity = quantity

    def __str__(self):
        return (
            f"Order ID: {self.order_id} | Customer: {self.customer_name} | "
            f"Food: {self.food_item} | Quantity: {self.quantity}"
        )

File: Order.java
/**
 * Order.java
 *
 * Represents a single food delivery order with core order details.
 */
public class Order {
    private String orderId;
    private String customerName;
    private String foodItem;
    private int quantity;

    public Order(String orderId, String customerName, String foodItem, int quantity) {
        this.orderId = orderId;
        this.customerName = customerName;
        this.foodItem = foodItem;
        this.quantity = quantity;
    }

    public String getOrderId() {
        return orderId;
    }

    public String getCustomerName() {
        return customerName;
    }

    public String getFoodItem() {
        return foodItem;
    }

    public int getQuantity() {
        return quantity;
    }

    @Override
    public String toString() {
        return String.format("Order ID: %s | Customer: %s | Food: %s | Qty: %d",
                orderId, customerName, foodItem, quantity);
    }
}
